### ISSUE TITLE
Allow Zenith Tower Modifier Notification To Be More Transparent

### ISSUE THREAD

**Issue by SplitPSoup** [o5k/tetrio-issues#1349]
### Checklist

- [X] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

When I play Zenith Tower and approach the top of the board when a new modifier (i.e. more permanent lines, more garbage, etc.), the notification obscures my vision, which 9 times out of 10 causes me to top out as I cannot see where to place my pieces due to how opaque the message is, and how long the message persists on my screen.

### Describe the solution you'd like

Create a slider in the settings to change the transparency of the notifications with one end being that the notifications do not show up at all, and the other end being that they are 100% opaque.

### Describe alternatives you've considered

Possible alternatives could be altering the size of the board in Zenith Tower so that messages on the top of the screen do not overlap with the board, creating a toggle to turn on or off the messages all together, and possibly moving the messages ot one of the sides of the screen.

**Comment by BubbleBlackTea** [comment]
Yes, I've had the same problem, it's really annoying. The fatigue sets notification appears when I'm about to lose and it blocks my vision so I can't see where to put the pieces.
